tor_core/initialize.py

```diff
@@ -197,19 +193,15 @@ def populate_subreddit_lists(config):
         config.subreddits_domain_filter_bypass
     )
     logging.debug(
-        'Retrieved subreddits that bypass the domain filter: {}'.format(
-            config.subreddits_domain_filter_bypass
-        )
+        f'Retrieved subreddits that bypass the domain filter: {config.subreddits_domain_filter_bypass}'
```

Comment:
I just realized that we don't actually have code standards in this repo. While I fix that, could you make sure that none of the lines changed in these PRs are longer than 80 characters? Archaic, I know, but it really does make a difference when viewing multiple files on the same screen.

Reply:
Gosh darn it @itsthejoker that's what I was doing

Reply:
@itsthejoker ok, sure